internal/migrations/migrations.go

```diff
@@ -15,6 +15,7 @@ func all() []*gormigrate.Migration {
 	allMigrations := []*gormigrate.Migration{
 		changeOverridesToText(),
 		changeImageSSHKeyToText(),
+		nonNilDiskEligibility(),
```

Comment:
Ideally the name would say what the migration does. So maybe something like `populateDiskEligibility` would be better.